cf-agent/verify_users_pam.c

```diff
@@ -793,6 +796,55 @@ static bool SetAccountLockExpiration(const char *puser, bool lock)
 
     return true;
 }
+#endif
+
+#ifdef HAVE_PW
+static bool SetAccountLockExpirationUsingPw(const char *puser, bool lock)
```

Comment:
Again try to avoid all the code duplication (copy-pasting) by having a shared helper function.